examples/template/template.js

```diff
@@ -8,10 +8,14 @@ export const options = {
 };
 
 const endpoint = __ENV.ENDPOINT || "otel-collector:4317"
+const orgid = __ENV.TEMPO_X_SCOPE_ORGID || "test"
```

Comment:
Should we include k6 in the tenant name by default?

Reply:
yeah, good idea. let me update it.